### PR TITLE
super-slicer-latest: 2.4.58.3 -> 2.4.58.5

### PR DESCRIPTION
**Pull request by Moredread**
###### Description of changes

Note that without PR https://github.com/NixOS/nixpkgs/pull/194021 

Changelog:

https://github.com/supermerill/SuperSlicer/releases/tag/2.4.58.5
https://github.com/supermerill/SuperSlicer/releases/tag/2.4.58.4

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/prusa-slicer/super-slicer.nix [hunk 1]
```diff
@@ -48,6 +48,10 @@ let
       "-DSLIC3R_BUILD_TESTS=0"
     ];
 
+    buildInputs = super.buildInputs ++ [
+      libspnav
```
Comment (cab404) [inline pkgs/applications/misc/prusa-slicer/super-slicer.nix:52]: wow Merill added spacemouse support?

Comment (Moredread) [reply pkgs/applications/misc/prusa-slicer/super-slicer.nix:52]: looks like it :) sadly it doesn't compile due to boost issues. I hope I find a working patch.

**Comment by hesiod** [comment]
@Moredread Would you mind rebasing to a recent master version? I think everything should build correctly, but there are merge conflicts at the moment.

**Comment by cab404** [comment]
Have anyone been successful setting up a development environment for it?

**Comment by hesiod** [comment]
@cab404 It worked fine when I tried it in a local commit. Did you have any issues?

**Comment by Moredread** [comment]
Sorry, I've rebased the commit, but hadn't time to build it yet. If you should have success testing it please let me know. :)

File: pkgs/applications/misc/prusa-slicer/super-slicer.nix [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{ lib, fetchFromGitHub, fetchpatch, makeDesktopItem, prusa-slicer }:
+{ lib, fetchFromGitHub, fetchpatch, makeDesktopItem, prusa-slicer, wxGTK31-gtk3, libspnav }:
```
Comment (hesiod) [inline pkgs/applications/misc/prusa-slicer/super-slicer.nix:1]:
```suggestion
{ lib, fetchFromGitHub, fetchpatch, makeDesktopItem, prusa-slicer, libspnav }:
```
This should be enough to fix the eval issues

**Comment by hesiod** [comment]
@Moredread I've taken the liberty of taking over the update commit in #225817 (also adding the beta version), I hope you don't mind :)

**Pull request closed without merging**